docs/6.-Engineering/Quality-Standards/Endpoint-Naming-Conventions.md

```diff
@@ -56,7 +56,7 @@ Wherever possible, a `GET` request should always be used for _reading_ data. If
     * `?collections=A,B` to filter by collection A or collection B
     * `?status=InComplete,ForReview` to filter by assets where the status is Incomplete for For Review
 
-A `POST` request should only ever be used in the case where the maximum length of a query string is insufficient to supply the complexity of the parameters.
+A `POST` request should only ever be used in the case where the maximum length of a query string is insufficient to supply the complexity of the parameters. A caveat for using `POST` is to send sensitive data, as POST data doesn't get cached or stored unlike query parameters which could potentially be cached or saved in a log, which can has better security implementation. 
```

Comment:
which can has better security implementation.

This part of the sentence doesn't really make sense

Reply:
Apologies, never really been that good at English. Hopefully have updated it to make sense.

Reply:
The sentence still reads like the query param has better security implementation rather than the post.

How about? 

_A caveat for the previous rule is using `POST` while sending sensitive data, as POST data doesn't get cached or stored. When using query parameters these could potentially be cached or saved in a log, so using a POST has better security implications._

Reply:
That sounds good. I have adjusted it slightly and updated. Made it more obvious its POST vs GET request in this scenario.

Reply:
Happy with this now